components/collection/drop/GenerativePreview.vue

```diff
@@ -37,7 +37,7 @@
         rel="nofollow noopener noreferrer"
         target="_blank"
         role="link">
-        {{ $t('Img') }}
+        {{ $t('img') }}
```

Comment:
translation does not exist?

Reply:
nope, had to add it 
![CleanShot 2023-12-27 at 10 36 00@2x](https://github.com/kodadot/nft-gallery/assets/44554284/ce7573e6-fa9d-4679-b0cd-b36a5fc96342)